spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -216,7 +230,7 @@ public void listen2(@Payload String foo, @Header(KafkaHeaders.PARTITION_ID) int
 			this.latch2.countDown();
 		}
 
-		@KafkaListener(id = "baz", topicPartitions = @TopicPartition(topic = "annotated3", partition = "0"))
+		@KafkaListener(id = "baz", topicPartitions = @TopicPartition(topic = "annotated3", partitions = "0"))
```

Comment:
Maybe `topic` in the `@TopicPatition` has to be required, not `optional` with `default ""`?
```
10:49:02.070 TRACE [bar-0-kafka-1][org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer] Polling...
10:49:02.113 WARN  [baz-0-kafka-1][org.apache.kafka.clients.NetworkClient] Error while fetching metadata with correlation id 138 : {=INVALID_TOPIC_EXCEPTION}
```